packages/oui-modal/README.md

```diff
@@ -47,4 +47,4 @@
 | `primary-label`     | string   | @?      | yes              |                        |                     | confirmation label                               |
 | `secondary-action`  | function | &?      |                  |                        |                     | cancellation callback                            |
 | `secondary-label`   | string   | @?      | yes              |                        |                     | cancellation label                               |
-| `on-close`          | function | &?      |                  |                        |                     | close callback                                   |
+| `on-dismiss`        | function | &?      |                  |                        |                     | close callback                                   |
```

Comment:
to keep the same vocabulary you should change the "close callback" sentence below too

Reply:
ok